src/main.c

```diff
@@ -711,6 +711,8 @@ static int databases_load() {
 
 static void hamlib_init() {
 
+    rig_set_debug(RIG_DEBUG_NONE);
+
```

Comment:
Maybe better in hamlib_init(). Not sure if the details of setting hamlibs  debug level is a responsibility for main().

Reply:
Did you mean `init_tlf_rig()`?
It is not called if trx control is disabled.
(`rig_set_debug` is already in `hamlib_init()`)

Reply:
I think I was reading the comments in wrong order, sri. So is this fine?

Reply:
No, sri. Was my fault. I thought you put it back to main(). It is fine at the start of hamlib_init().